package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dashevo/dashcore-lib",
-  "version": "1.0.0-alpha.23",
+  "version": "1.0.0-alpha.24",
```

Comment:
This should probably also be in `package-lock.json` as well, right?

Reply:
yes, updated!